lib/active_shipping/carriers/ups.rb

```diff
@@ -1201,5 +1201,9 @@ def package_level_delivery_confirmation?(origin, destination)
       origin.country_code == destination.country_code ||
       [['US','PR'], ['PR','US']].include?([origin,destination].map(&:country_code))
     end
+
+    def patch_country_code(country_code)
```

Comment:
This really doesn't communicate what it does. `patch` is so generic, can we just call this `mapped_country_code` or something? #NamingIsHard

e.g.

```
COUNTRY_MAPPING = {
  'XK' => 'KV'
}.freeze

...

def mapped_country_code(country_code)
{
  COUNTRY_MAPPING[country_code].presence || country_code
}
```